vault/core.go

```diff
@@ -1732,6 +1727,12 @@ func (c *Core) postUnseal() (retErr error) {
 	c.metricsCh = make(chan struct{})
 	go c.emitMetrics(c.metricsCh)
 
+	// Enable the cache
```

Comment:
Any reason not to enable the cache earlier?  I think we would want to cache some of these entries.

Reply:
Just that most of these things are unlikely to be read out of cache during normal operation. I don't think it matters much though, i'll move it back up for now